### PR TITLE
snakeviz 2.2.0 (new formula)

### PR DESCRIPTION
**Pull request by rtau**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/s/snakeviz.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    url "https://files.pythonhosted.org/packages/48/64/679260ca0c3742e2236c693dc6c34fb8b153c14c21d2aa2077c5a01924d6/tornado-6.3.3.tar.gz"
+    sha256 "e7d8db41c0181c80d76c982aacc442c0783a2c54d6400fe028954201a2e032fe"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/snakeviz", "--version"
```
Comment (chenrui333) [inline Formula/s/snakeviz.rb:22]: need a more substantial test

**Comment by chenrui333** [comment]
@rtau maybe use py3.12?

**Comment by rtau** [comment]
> @rtau maybe use py3.12?

Sorry, will try that when have time. Maybe make this PR draft until it's ready for review again? I'd like to make more realistic test as well.

**Comment by chenrui333** [comment]
> > @rtau maybe use py3.12?
> 
> Sorry, will try that when have time. Maybe make this PR draft until it's ready for review again? I'd like to make more realistic test as well.

I just tested out, it works in my local. :)

**Review by chenrui333 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**